gtsam/geometry/Rot3.h

```diff
@@ -194,22 +194,24 @@ namespace gtsam {
 
     /**
      * Convert from axis/angle representation
-     * @param  axisw is the rotation axis, unit length
-     * @param   angle rotation angle
+     * @param  axis is the rotation axis, unit length
+     * @param  angle rotation angle
      * @return incremental rotation
      */
-    static Rot3 AxisAngle(const Point3& axis, double angle) {
+    static Rot3 AxisAngle(const Vector3& axis, double angle) {
```

Comment:
Changing type: not a good idea: Point3 and Vector3 are not always the same. See Compilation flags.

Reply:
Good point. Sorry for overlooking that.